.github/workflows/integration.yml

```diff
@@ -1,14 +1,16 @@
 name: bootc integration test
 on:
   pull_request:
-    branches: [main]
+    branches: [main]  
+  workflow_dispatch:
 
 jobs:
   build:
     strategy:
       matrix:
-        test_os: [fedora-41, fedora-42, fedora-43, centos-9]
-        test_runner: [ubuntu-latest, ubuntu-24.04-arm]
```

Comment:
We do not run arm build test in github action runner? No nested virt support in github arm runner.

Reply:
> No nested virt support in github arm runner.

:cry: 

> We can only keep x86_64 test running in github action, and run arm test on Packit.

Hmmm...maybe yeah that's one way to dice the split.

I mean now, in reality, very very very little of the code is architecture sensitive and it's not clear to me we need to run *all* tests across both architectures across *four operating system major versions*...but trimming down that matrix is a whole other topic.